.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -1,5 +1,6 @@
 name: Feature Request
 description: Got an idea for a feature that Gitea doesn't have currently?  Submit your idea here!
+labels: kind/feature 
```

Comment:
Maybe proposal as well?

Reply:
Well, the problem I have with that is:
What exactly is the difference between feature and proposal?
I've seen both used, but for me, they mean pretty much the same thing.
I mean, I can add it, but aren't these two labels simply duplicates? Or do I understand something wrong?

Reply:
I've added it now. However, I still would like to know the difference between `kind/proposal` and `kind/feature`.